tasks/main.yml

```diff
@@ -48,7 +55,7 @@
   template:
      src=unbound.conf.j2
      dest=/etc/unbound/unbound.conf
-     validate="/usr/sbin/unbound-checkconf %s"
+     #validate="/usr/sbin/unbound-checkconf %s"
```

Comment:
Have you find some trouble with the validation ?

Reply:
I think I found out why this was commented - "fatal error: config file /home/ec2-user/.ansible/tmp/ansible-tmp-1543938114.2980123-1210814837092/source is not inside chroot /etc/unbound"
I can't think of an way to fix this as Ansible is checking against the tmp location